balancer-js/examples/pools/composable-stable/create.ts

```diff
@@ -0,0 +1,203 @@
+import * as dotenv from 'dotenv';
```

Comment:
Can you tidy this file up. If I run it with line 142 uncommented I get "There's no event".